<launch>
    <arg name="serial_port" default="/dev/ttyACM0" />
    <arg name="baud_rate" default="115200" />
    <node pkg="uwb" type="uwb_serial_node" name="uwb_serial" output="screen">
        <param name="serial_port" value="$(arg serial_port)" />
        <param name="baud_rate" value="$(arg baud_rate)" />
    </node>
</launch>
